src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/boss_tomb_of_seven.cpp

```diff
@@ -26,6 +26,17 @@ enum Misc
     DATA_SKILLPOINT_MIN                           = 230
 };
 
+enum Gossip
+{
+    GOSSIP_TEXT_CONTINUE                          = 1828, // Continue...
+    GROOMREL_GOSSIP_OPTIONS                       = 1945, // Option 1 : Before quest(4083) accepted, option 0 after quest(4083) accepted
+    DOOMREL_START_COMBAT                          = 1947, // Your bondage is at an end, Doom'rel.  I challenge you!
+    SAY_DOOMREL_HELLO                             = 2601, // Our fate is the doom of all who face the Great Fire.
+    SAY_QUEST_ACCEPTED                            = 2604, // You wish to learn the old craft?  You wish to smelt dark iron?$B$BAppease me, $r.  Show me a sacrifice and I will consider it!
+    SAY_QUEST_COMPLETED                           = 2605, // Your will is strong, and your intent is clear.$B$BPerhaps you are worthy...
+    SAY_QUEST_COMPLETED_END                       = 2606, // You have shown me your desire, and have payed with precious stone.  I will teach you...
```

Comment:
I suggest using GOSSIP_ as the prefix for all gossips

Reply:
DOOMREL_START_COMBAT this is a say or gossip?

Reply:
The gossips are id 1828, 1945 and 1947. The rest are npc_text data.

Reply:
I updated them so the gossips starts with GOSSIP_
Because one of them was called GROOMREL_GOSSIP_OPTIONS i renamed it as GOSSIP_GROOMREL.